cmd_numchunks.lua

```diff
@@ -0,0 +1,22 @@
+function HandleNumChunksCommand(Split, Player)
+	-- List each world's chunk count into a table, sum the total chunk count:
+	local Response = {}
+	local Total = 0
+
+	local GetWorldChunks = function(World)
+		table.insert(Response, SendMessage(Player, World:GetName() .. ": " .. World:GetNumChunks() .. " chunks"))
+		Total = Total + World:GetNumChunks()
```

Comment:
This looks really weird, adding a `SendMessage` call to the table.
Anyway, you are aware that the two calls to `World:GetNumChunks()` could theoretically return different numbers, due to threading? Better call once and store the value in a temporary, so that the  sum is correct.

Reply:
I'll probably make a single SendMessage call for the whole response instead, and fix up the output format a bit.

Thanks for pointing out the GetNumChunk issue, it was an oversight when modifying the previous console variant of the command.